UPGRADING.md

```diff
@@ -1,6 +1,70 @@
 Upgrading Grape
 ===============
 
+### Upgrading to >= 1.3.4
+
+#### Reworking stream and file and un-deprecating stream like-objects
+
+Previously in 0.16 stream-like objects were deprecated. This release restores their functionality for use-cases other than file streaming.
+
+This release also renamed `file` to `sendfile` to better document it's purpose.
```

Comment:
it's -> its

We're deprecating `file`, so I would say "this release deprecated `file` in favor of `sendfile`"